<launch>
  <node name="interactXYTheta" pkg="mas_egraphs" type="interactXYTheta" output="screen" >
    <rosparam file="$(find mas_egraphs)/config/costmap_common_params.yaml" command="load" ns="global_costmap" />
    <rosparam file="$(find mas_egraphs)/config/global_costmap_params.yaml" command="load" />
  </node>
</launch>
